dev-games/box2d/patches/box2d-2.3.1.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/Box2D/CMakeLists.txt b/Box2D/CMakeLists.txt
+index a8ad550..17b3069 100644
+--- a/Box2D/CMakeLists.txt
++++ b/Box2D/CMakeLists.txt
+@@ -26,7 +26,8 @@ if(BOX2D_BUILD_EXAMPLES)
+ 
+   # Testbed and dependencies.
+   find_package(OpenGL REQUIRED)
++if(NOT HAIKU) # Haiku has glut, so no need for freeglut. Disable GLUI because it's not ported.
+   add_subdirectory(freeglut)
+   add_subdirectory(glui)
++endif(NOT HAIKU)
+-  add_subdirectory(Testbed)
```

Comment:
Don't delete this line; just put it in the NOT HAIKU section if needed. Also, indent this section properly.